index.html

```diff
@@ -186,6 +146,46 @@ <h4>Enterprise</h4>
         </div>
       </section>
 
+      <section class="asked-questions">
+        <div class="asked-questions__left">
+        
+          <details>
+            <summary>Frequently Asked Question 1<span>&#8250;</span></summary>
```

Comment:
it is not written using numbers words e.g. one, two, etc? change also the others

the unicode symbol you are using for this it has to be different from the others, the direction has to be towoards the bottom